### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by darkweak**

### CODE REVIEW

File: plugins/caddy/go.mod [hunk 1]
```diff
@@ -1,9 +1,9 @@
-module github.com/darkweak/souin/plugins/caddy
+module github.com/caddyserver/cache-handler
```
Comment (polarathene) [inline plugins/caddy/go.mod:1]: What's happening here? Are you no longer intending to have a separate Souin caddy module for advanced use, only supporting the `caddyserver/cache-handler` module?

File: plugins/caddy/README.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ This is a distributed HTTP cache module for Caddy based on [Souin](https://githu
 
  * Supports most HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html) (see the TODO section for limitations)
  * Sets [the `Cache-Status` HTTP Response Header](https://httpwg.org/http-extensions/draft-ietf-httpbis-cache-header.html)
- * REST API to be able to purge the cache and list stored keys.
- * Embedded (or not) distributed cache can be used and a non-distributed one is supported too.
+ * REST API to purge the cache and list stored resources.
+ * Builtin support for distributed cache.
```
Comment (polarathene) [inline plugins/caddy/README.md:11]: Are you completely removing the support for YKey and flexible cache providers?

File: plugins/caddy/go.mod (hunk 1 above)
Comment (darkweak) [reply plugins/caddy/go.mod:1]: Reverted 2 days ago ;)

File: plugins/caddy/README.md (hunk 2 above)
Comment (darkweak) [reply plugins/caddy/README.md:11]: I'm working on a fully specified system called `Surrogate-keys` and make a RFC for this.

**Pull request merged**